.github/workflows/build-multiarch.yaml

```diff
@@ -61,7 +66,8 @@ jobs:
 
       - uses: docker/setup-buildx-action@v3
 
-      - id: calculate-image-tags
+      - name: Calculate Image Tags
```

Comment:
optional: no need to set name unless it's providing information that's not already clear from the ID

Reply:
I set the name here because otherwise in the run log, it comes out like this:
<img width="488" alt="image" src="https://github.com/alphagov/govuk-ruby-images/assets/68009085/f8cef12c-daec-4339-b9bb-d9ab41557c78">
...which to me, just looks yucky.